### PR TITLE
feat(contrib): add nacos unit test and remove unused function

### PR DESCRIPTION
**Pull request by wcvb13**
**Description (what this PR does / why we need it):**
add nacos unit test

**Which issue(s) this PR fixes (resolves / be part of):**
part of [#2135](https://github.com/go-kratos/kratos/issues/2135)

Other special notes for the reviewers:

### CODE REVIEW

File: contrib/config/nacos/config.go [hunk 1]
```diff
@@ -42,34 +32,6 @@ func WithDataID(dataID string) Option {
...
-	return func(o *options) {
-		o.logLevel = logLevel
-	}
-}
-
-// WithTimeout With nacos config timeout.
-func WithTimeout(time time.Duration) Option {
-	return func(o *options) {
-		o.timeoutMs = uint64(time.Milliseconds())
-	}
```
Comment (Cluas) [inline contrib/config/nacos/config.go:70]: This is the option api exposed to the user, bring a bit of your own thinking and don't rely on IDE hints.

Comment (Cluas) [reply contrib/config/nacos/config.go:70]: A better way is to add the relevant Option tests, which will make your linter happy.

Comment (wcvb13) [reply contrib/config/nacos/config.go:70]: This option is a parameter of the nacos client, the nacos client is created by the user and passed into newsource function, so this option is unused

Comment (wcvb13) [reply contrib/config/nacos/config.go:70]: If you want to use this option, it will need to re-implement the newsource method

Comment (Cluas) [reply contrib/config/nacos/config.go:70]: Sorry, I misunderstood, it turns out that the option defined here is not used at all😅

**Review by tonybase — APPROVED** [review]

**Review by shenqidebaozi — APPROVED** [review]

**Pull request merged**